src/matchbox/core.cljc

```diff
@@ -506,6 +506,12 @@
 
 ;; onAuth and offAuth are not wrapped yet
 
+(defn onAuth [ref cb]
+  (.onAuth ref (wrap-auth-cb cb)))
+
+(defn offAuth [ref cb]
+  (.offAuth ref cb))
```

Comment:
What payload does this callback receive? Should we not also wrap it to return Clojure data instead of a raw JS object?